client/Prompts/SpellPrompt.tsx

```diff
@@ -19,6 +19,7 @@ export function SpellPrompt(
       </>
     ),
     initialValues: {},
+    className: "prompt-spell",
```

Comment:
I don't think you need to update the whole Prompt API for this- can you just wrap the `children` in a div with this className on line 16?

Reply:
I was trying to avoid introducing a new layer into the hierarchy, just in case that broke something I didn't see coming.

![image](https://user-images.githubusercontent.com/85644845/121689163-ea969700-ca89-11eb-9bf0-9612b0be2c23.png)

Plus, by being able to add custom class names to the parent, we have the ability in the future to apply conditional styling to the whole box (e.g. concentration prompts with an alert coloration to catch your attention in the midst of running a combat).

Still, I can definitely make a wrapper around the children work, though, if you'd prefer, though it takes a little more on the CSS side to make things play nice.

I have to make the `.prompt-spell` repeat some of `.prompt`'s style definitions...

```css
.prompt-spell {
    flex-direction: row; /* or... flex-direction: inherit; */
    align-items: flex-start; /* or... use height: fit-content on the button */
}
```

But the end result is pretty much identical on the visual front.

![image](https://user-images.githubusercontent.com/85644845/121690159-19f9d380-ca8b-11eb-98a5-50d2801770b2.png)

Let me know if you'd prefer to do it this way.

Reply:
I think I prefer the CSS only approach here that you've described. Thanks!

Reply:
Sorry for sitting on this for a week. I've pushed this adjustment into the fork (a1d57f5be5fbf6548a76bf7f384756534e1c1f8a).